app/lib/services/get_an_identity/user_updater.rb

```diff
@@ -12,9 +12,9 @@ def initialize(user:)
       end
 
       def call
-        get_an_identity_user = ::GetAnIdentity::User.find(user.get_an_identity_id)
+        get_an_identity_user = ::GetAnIdentity::External::User.find(user.get_an_identity_id)
```

Comment:
Thinking about this now i think we should have external outside the GAI namespace? That way keeps it very clear and seperate like Peter suggested.

Reply:
Good point, swapped around